lib/beaker/options/parser.rb

```diff
@@ -90,7 +90,9 @@ def file_list(paths)
       # @return nil
       # @api public
       def resolve_symlinks!
-        @options[:hosts_file] = File.realpath(@options[:hosts_file]) if @options[:hosts_file]
+        if @options[:hosts_file] && !@options[:hosts_file_generated]
```

Comment:
So we weren't able to hide the fact that the file wasn't pre-generated and had to introduce this "option" to detect that?